internal/app/set/main_test.go

```diff
@@ -1,19 +1,42 @@
 package set
 
 import (
+	"fmt"
 	"testing"
 )
 
 func TestSetSerializeCommand(t *testing.T) {
```

Comment:
Add more test cases, at least a few for each of the `SerializeSetCommand` and the `SerializeSetDataBlock` functions. 1 test case for each is certainly not enough.